lib/config/options/index.ts

```diff
@@ -3095,6 +3095,46 @@ const options: RenovateOptions[] = [
     default: 90,
     globalOnly: true,
   },
+  {
+    name: 'dockerMaxPages',
+    description:
+      'By default, Renovate fetches up to 20 pages of Docker tags from registries, but this maximum can be adjusted by setting this option.',
+    type: 'integer',
+    default: 20,
+    globalOnly: true,
+  },
+  {
+    name: 'gitlabAutoMergeableCheckAttempts',
```

Comment:
@rarkins @secustor I think we should unify this under a generic platform options object, so we've a single config option with known values. not is best time to do it.

WDYT?

Reply:
I'm not sure I understand. Do you mean nesting it inside a `platformOptions` object? Does it give any advantage other than making sure all possible ones are located together within config?

Reply:
those are also easier findable in docs, as they're under same parent headline and not mixed all over the config options.

Reply:
Create another issue: https://github.com/renovatebot/renovate/issues/29716